src/views/tickets/Screenshot.js

```diff
@@ -74,8 +72,14 @@ export default class Screenshot extends Component {
             />
           </Slide>
         );
-        carouselTemp.push(carouselImg);
-        this.setState({ carousel: carouselTemp });
+
+        this.carousel.push(carouselImg);
+        getArtifact(
+          this.props.ticketID,
+          'recognize-' + filename.slice(0, -4) + '.ocr'
```

Comment:
File extension is not necessarily three characters (e.g. `.tiff`, `.jpeg`) - can [use regex or string manipulation](https://stackoverflow.com/questions/4250364/how-to-trim-a-file-extension-from-a-string-in-javascript) to not rely on a set length.

Reply:
Sure thing. This is [the list](https://github.com/arthurvr/image-extensions/blob/master/image-extensions.json) of extensions the honeyclient recognizes.

Reply:
On this note, do we need to even do this to check for a valid filename? Currently, I'm thinking that we could split the filename by ".", and just use the latter half (the extension). The honeyclient will recognize that this is an unsupported filetype and just ignore it no?

Reply:
I'm leaving this open because still not sure how we should implement this

Reply:
We could do a whitelist instead of a blacklist here. Split on the `.` like you suggest, then say `.yara` and `.js` are filtered out.

Reply:
Also this has to do with the OCR so if we are removing that functionality we won't have to decide on this